src/Illuminate/Support/Collection.php

```diff
@@ -1890,6 +1890,28 @@ public function count()
         return count($this->items);
     }
 
+    /**
+     * Count the number of items in the collection by some predicate.
+     *
+     * @param callable|null
+     * @return static
+     */
+    public function countBy($predicate = null)
+    {
+        if (is_null($predicate)) {
+            $predicate = function ($val, $key) {
```

Comment:
```suggestion
            $predicate = function ($val) {
```

You aren't utilizing the `$key` so it doesn't need to be set.